src/style/block/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
Should be selector `*` with with property:
```
* {
  box-sizing: border-box;
}
```

https://github.com/mate-academy/layout_product-cards/pull/2597#discussion_r1113053739